scala/scala_cross_version.bzl

```diff
@@ -5,23 +5,71 @@
 # You may obtain a copy of the License at
 #
 #    http://www.apache.org/licenses/LICENSE-2.0
-#
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+load(
+    "@io_bazel_rules_scala//scala:scala_maven_import_external.bzl",
+    _scala_maven_import_external = "scala_maven_import_external")
 """Helper functions for Scala cross-version support. Encapsulates the logic
 of abstracting over Scala major version (2.11, 2.12, etc) for dependency
 resolution."""
 
-def scala_version():
+def default_scala_version():
   """return the scala version for use in maven coordinates"""
-  return "2.11"
+  return "2.11.12"
 
-def scala_mvn_artifact(artifact):
+def default_scala_version_jar_shas():
+  return {
+      "scala_compiler": "3e892546b72ab547cb77de4d840bcfd05c853e73390fed7370a8f19acb0735a0",
+      "scala_library": "0b3d6fd42958ee98715ba2ec5fe221f4ca1e694d7c981b0ae0cd68e97baf6dce",
+      "scala_reflect": "6ba385b450a6311a15c918cf8688b9af9327c6104f0ecbd35933cfcd3095fe04",
+  }
+
+def scala_mvn_artifact(artifact, major_scala_version):
```

Comment:
do you want to give a `major_scala_version = extract_major_version(default_scala_version())` variant to avoid breaking people?